plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -616,17 +616,22 @@ fu_ccgx_dmc_device_attach(FuDevice *device, FuProgress *progress, GError **error
 	if (fu_device_get_update_state(self) != FWUPD_UPDATE_STATE_SUCCESS)
 		return TRUE;
 
+	if (manual_replug == FALSE)
+		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
+
 	if (self->update_model == DMC_UPDATE_MODEL_DOWNLOAD_TRIGGER) {
 		if (self->trigger_code > 0) {
 			if (!fu_ccgx_dmc_device_send_download_trigger(self,
 								      self->trigger_code,
 								      error)) {
+				fu_device_remove_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
```

Comment:
presumably this isn't needed if you move it to lower

Reply:
Don't  I need remove FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG flag when return FALSE, if I set FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG flag before?